### DISCUSSION TITLE
All images links are broken on dev.chaskiq.io ?

### DISCUSSION THREAD

**Discussion by pierrecorsini** [chaskiq/chaskiq#987] [Q&A]
All the images in the pages i.e. `https://dev.chaskiq.io/third-party-integration/facebook-messenger` are not displayed.

Is there any way to find the images ?

![image](https://github.com/chaskiq/chaskiq/assets/50719398/5afb81ca-0413-4d45-851e-9a8efe24b5dc)

**Comment by michelson** [comment]
Thanks for the heads up. fixed in main branch, deployed.
